assets/scss/styles.scss

```diff
@@ -119,76 +125,54 @@ h1 {
 }
 .bed-count-label {
   @include core-19;
-  font-size: 19px;
   line-height: 0.9;
   display: inline-block;
   white-space: normal;
 }
 
 /* Stats page */
-#stats {
+.stat_data {
+  position: relative;
+  .centre_updated {
+    @include updated_timestamp;
+  }
   table {
+    @include media(320px, 640px) {
+      margin: 0 0 2em;
+      width: 100%;
+    }
+    margin: 0 0 10% 15%;
     border-collapse: collapse;
     border-spacing: 0;
-    width: 100%;
-    margin-bottom: 50px;
+    box-sizing: border-box;
+    width: 85%;
     thead {
       tr {
         th,
         td {
-          padding-top: 0;
-          padding-bottom: 16px;
+          padding: 0.75em 1.25em 0.5625em 0;
           white-space: nowrap;
           vertical-align: bottom;
-        }
-        th:last-child {
-          text-align: right;
+          &.numeric {
+            @include bold-19($tabular-numbers: true);
+            text-align: right;
+          }
         }
       }
     }
-    thead,
     tbody {
+      background-color: $background-color;
       tr {
-        th {
-          &.numeric {
-            text-align: left;
-          }
-        }
-        td {
-          &.numeric {
-            font-family: "ntatabularnumbers", "nta", Arial, sans-serif;
-            font-size: 14px;
-            line-height: 1.1428571429;
-            font-weight: 400;
-            text-transform: none;
-            @media (min-width: 641px) {
-              font-size: 16px;
-              line-height: 1.25;
-            }
-          }
-        }
         th,
         td {
           @include core-19;
           padding: 0.75em 1.25em 0.5625em 0;
           text-align: left;
-          color: #0b0c0c;
-          border-bottom: 1px solid #6f777b;
           width: 25%;
```

Comment:
this whole table + page layout will cause wrapping+cropping issues if I for example had a table that looked like:

header1 | header2| header3
---|---|---
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa | aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa | aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa | aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa | aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

or someone was using a browser that increased the font-size to the equivalent